boards/arm/npcx4m8f_evb/doc/index.rst

```diff
@@ -0,0 +1,131 @@
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc controller                      |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c port/controller                 |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| PM        | on-chip    | power management                    |
++-----------+------------+-------------------------------------+
+| PSL       | on-chip    | power switch logic                  |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pulse width modulator               |
++-----------+------------+-------------------------------------+
+| TACH      | on-chip    | tachometer sensor                   |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| WDT       | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the defconfig file:
+``boards/arm/npcx4m8f_evb/npcx4m8f_evb_defconfig``
+
+
+Connections and IOs
+===================
+
+Nuvoton to provide the schematic for this board.
+
+System Clock
+============
+
+The NPCX4M8F MCU is configured to use the 120Mhz internal oscillator with the
+on-chip PLL to generate a resulting EC clock rate of 15 MHz. See Processor clock
+control register (chapter 4 in user manual)
+
+Serial Port
+===========
+
+UART1 is configured for serial logs.
+
+Programming and Debugging
+*************************
+
+This board comes with a Cortex ETM port which facilitates tracing and debugging
+using a single physical connection. In addition, it comes with sockets for
+JTAG-only sessions.
+
+Flashing
+========
+
+If the correct headers are installed, this board supports both J-TAG and also
+the ChromiumOS servo.
+
+To flash using Servo V2, μServo, or Servo V4 (CCD), see the
+`Chromium EC Flashing Documentation`_ for more information.
+
+To flash with J-TAG, install the drivers for your programmer, for example:
+SEGGER J-link's drivers are at https://www.segger.com/downloads/jlink/
+
+The openocd from Zephyr SDK 0.14.2 doesn't include npcx support, so build openocd from source.::
```

Comment:
Hey does this still apply? We are at 0.16.1, may be worth dropping a version number and keep it generic, then if this will get supported this can be changed referring to the first zephyr SDK that supports it.

Reply:
Oops. It's my mistake not to mention this. Since npcx4 series introduces new FIU hardware, Nuvoton submitted the CL in openocd's repo for programming internal flash recently. Please refer to [CL 7794](https://review.openocd.org/c/openocd/+/7794) for more details. I believe that it will be supported in later Zephyr SDKs.

Reply:
The lastest CL in [openocd](https://github.com/zephyrproject-rtos/openocd) used for zephyr-sdk is [on Apr 22](https://github.com/zephyrproject-rtos/openocd/commit/10e3679dc992a53f3f55dd19851da1679c94c078). It might take a while to support this board.

I will prepare the other PR for the npcx boards which have supported openocd.